src/node/zlib.ts

```diff
@@ -100,12 +213,122 @@ export {
   brotliDecompressSync,
   brotliCompress,
   brotliCompressSync,
+
+  // NodeJS also exports all constants directly under zlib, but this is deprecated
```

Comment:
@jasnell afaik, there isn't any other alternative way to expose this, but any idea?

Reply:
Well... you could move all of the constants here into a separate `internal/zlib-constants` import as named exports only... then do

```js
import * as zlibConstants from 'node-internal:zlib-constants';

export const constants = { ...zlibConstants };
export * from 'node-internal:zlib-constants';
```

But might not be worth the extra hassle. I'd just go with this for now.

Reply:
JavaScript is a weird language.